bundles/org.openhab.core.config.discovery.addon.ip/src/main/java/org/openhab/core/config/discovery/addon/ip/IpAddonFinder.java

```diff
@@ -322,6 +282,120 @@ private void scan() {
         logger.trace("IpAddonFinder::scan completed");
     }
 
+    private void scanBroadcast(AddonInfo candidate, String request, String response, int timeoutMs, int destPort) {
+        if (request.isEmpty()) {
+            logger.warn("{}: match-property request \"{}\" is unknown", candidate.getUID(), TYPE_IP_BROADCAST);
+            return;
+        }
+        if (response.isEmpty()) {
+            logger.warn("{}: match-property response \"{}\" is unknown", candidate.getUID(), TYPE_IP_BROADCAST);
+            return;
+        }
+        try (DatagramSocket socket = new DatagramSocket()) {
+            Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
+            while (interfaces.hasMoreElements()) {
+                @Nullable
+                NetworkInterface networkInterface = interfaces.nextElement();
+                if (networkInterface.isLoopback() || !networkInterface.isUp()) {
+                    continue;
+                }
+                for (InterfaceAddress interfaceAddress : networkInterface.getInterfaceAddresses()) {
+                    if (interfaceAddress.getBroadcast() == null) {
+                        continue;
+                    }
+                    socket.setBroadcast(true);
```

Comment:
Rather than using the standard broadcast address, you could use the configurable one from network config. This could limit network traffic.